pkg/instancetype/instancetype.go

```diff
@@ -386,6 +386,10 @@ func checkCPUPreferenceRequirements(instancetypeSpec *instancetypev1beta1.Virtua
 		if vmiSpec.Domain.CPU.Sockets < preferenceSpec.Requirements.CPU.Guest {
 			return cpuField.Child("sockets"), fmt.Errorf(InsufficientVMCPUResourcesErrorFmt, vmiSpec.Domain.CPU.Sockets, preferenceSpec.Requirements.CPU.Guest, "sockets")
 		}
+	case instancetypev1beta1.PreferSpread:
+		if vmiSpec.Domain.CPU.Cores+vmiSpec.Domain.CPU.Sockets < preferenceSpec.Requirements.CPU.Guest {
```

Comment:
Can you stash `vmiSpec.Domain.CPU.Cores+vmiSpec.Domain.CPU.Sockets` in a meaningful variable as it's reused below?